doc/sailfish-webview-qmozcontext.h

```diff
@@ -0,0 +1,41 @@
+/****************************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+** All rights reserved.
+**
+****************************************************************************************/
+
+#include <QObject>
+
+// Used only for documentation purposes
```

Comment:
Instead of "Used only for documentation purposes" maybe "Dummy declaration for documentation purposes only".

Reply:
Thanks; I've changed it.